src/tea_http.res

```diff
@@ -69,12 +69,12 @@ let expectStringResponse = func => {
     ({body, _}) =>
       switch body {
       | TextResponse(s) => func(s)
-      | _ => Tea_result.Error("Non-text response returned")
+      | _ => Belt.Result.Error("Non-text response returned")
       },
   )
 }
 
-let expectString = expectStringResponse(resString => Tea_result.Ok(resString))
+let expectString = expectStringResponse(resString => Belt.Result.Ok(resString))
```

Comment:
Instead of using `Belt.Result.Ok`, you should be able to use just `Ok`, right?

Reply:
Yes it does work and this applies to Error as well.
for Tea_result.res, it is still used in some places that is why I didn't remove it.